index.js

```diff
@@ -13,9 +13,9 @@ try {
  * Defer.
  */
 
-var defer = typeof process === 'undefined' || typeof process !== 'function'
-  ? function(fn){ setTimeout(fn); }
-  : process.nextTick;
+var defer = typeof process !== 'undefined' && typeof process.nextTick === 'function'
```

Comment:
> I already gave you an example: In a web browser, since process has no meaning, it can be any value if someone is using that for something. That's the whole purpose of this line.

in a web browser, `process` will be `undefined`, not `null`. they have very different meanings in javascript.

if someone sets `process = null`, i'd expect many things/modules to break, and do not think it's practical for such defensive programming.

for clarity, please refer to the screenshot in https://github.com/visionmedia/batch/pull/35#discussion_r116853346 or paste the statement (`typeof process !== 'undefined' && typeof process.nextTick === 'function'`) into your browser's console. even though `process === undefined`, the statement will evaluate to `false`.